### PR TITLE
Early baf generation

### PR DESCRIPTION
**Pull request by tedsharpe**

### CODE REVIEW

**Comment by tedsharpe** [comment]
VCF_QC 1000G_Sudmant_ALL callset_benchmarking
[VCF_QC.freq_distributions.merged.pdf](https://github.com/broadinstitute/gatk-sv/files/8825940/VCF_QC.freq_distributions.merged.pdf)
VCF_QC genotype_distributions
[VCF_QC.size_distributions.merged.pdf](https://github.com/broadinstitute/gatk-sv/files/8825941/VCF_QC
[VCF_QC.SV_counts.merged.pdf](https://github.com/broadinstitute/gatk-sv/files/8825942/VCF_QC.SV_counts.merged.pdf)
.size_dist
VCF_QC SV_per_genome
ributions.merged.pdf)

**Comment by tedsharpe** [comment]
[svtest.txt](https://github.com/broadinstitute/gatk-sv/files/8826043/svtest.txt)

**Comment by tedsharpe** [comment]
This is not quite ready to go.  I need to make a few updates due to the LD --> SD renaming.

File: wdl/BatchEvidenceMerging.wdl [hunk 1]
```diff
@@ -2,289 +2,187 @@ version 1.0
...
-           input:
-             file = select_first([BAF_files_value[i]]),
-             sample_id = samples[i],
-             output_name = "BAF_GatherBatchEvidence.~{samples[i]}.txt.gz",
-             sample_column_index = 4,
-             sv_base_mini_docker = sv_base_mini_docker,
-             runtime_attr_override = runtime_attr_set_sample
-        }
-      }
-    }
```
Comment (mwalker174) [inline wdl/BatchEvidenceMerging.wdl:48]: Deleting these from the main pipeline is definitely the right move at this point. However, we still need a means of changing sample IDs in evidence files for some projects. Can you add a new `SetSampleIdLegacy` workflow that takes in an SR, PE, and BAF file for a single sample and converts the sample ID in each file?

Also can you update the `Sample ID requirements` section of the README and include a pointer to the `SetSampleIdLegacy` workflow for evidence files generated with previous versions of the pipeline?

**Review by mwalker174 — CHANGES_REQUESTED** [review]
@tedsharpe it's like music to my ears. Benchmarks check out, just a few requests.

Thanks for updating the README. I think there are a few spots that need updating still - searching "gvcf" shows there are a few references that can be deleted.

Edit: also, the LD files were a bit larger than I was hoping for at ~300MB. I think it's tenable for now. I have a new sites list generated from gnomAD that's almost ready and might be able to cut that in half, but let's save it for another PR.

**Comment by mwalker174** [comment]
> 1. Have to wait until gatk/tws_bafFixes is available as a docker image (or I could make one) and update dockers.json.

Yes you could move the nightly (https://hub.docker.com/r/broadinstitute/gatk-nightly) after you merge onto GCR and use that.

> 2. I can't find superfluous mentions of gcnv as mentioned in the 1st review comment:  the references in wdl inputs appear to me to be downstream of BAF generation where we're using the pre-built random forest model to evaluate new samples.  @mwalker174: Can you point me to an example or two and I'll try to clean up the rest by analogy.

Was just pointing out `gvcf` (since those are no longer needed) not `gcnv`.

**Comment by tedsharpe** [comment]
Updated dockers.json to point to a gatk image that has the is consistent with current head of master (having just checked in that small bug fix).  Eliminated gvcf references.  I think we're good to go.

**Review by mwalker174 — APPROVED** [review]
Thank you @tedsharpe I think it's almost ready! I still see some gVCF occurrences in the README if you could just take those out, then good to merge.

**Pull request merged**